test/integration2/query/misc/additional.spec.js

```diff
@@ -135,13 +135,15 @@ describe('Additional', function () {
           let response;
           const stream = knex('accounts').limit(1).stream();
 
+          let count = 0;
           stream.on('data', (res) => {
+            count++;
             response = res;
           });
           stream.on('finish', () => {
-            return expect(response.callCount).to.equal(1);
+            expect(count).to.equal(1);
```

Comment:
this would still pass if finish is never invoked, wouldn't it? best approach would be to still await stream and then assert this count after that.

Reply:
I just realise that I can't await a stream unfortunaly ^^ https://stackoverflow.com/a/33599789/1535159

Reply:
Yeah, I was unsure it would work as expected, hence the suggestion to add more explicit assertions.

Reply:
I have a test that is valid on my machine but always fail (/home/runner/work/knex/knex/test/integration2/query/update/updates.spec.js:124:34) with transactions, do you have suggestions about how to fix it ?